### ISSUE TITLE
Update spacing tokens to use numbers

### ISSUE THREAD

**Issue by dcmouyard** [forumone/gesso#292]
The spacing tokens currently use letters for shorthand sizes.
```
  spacing:
    xxs: 4px
    xs: 8px
    sm: 16px
    md: 24px
    lg: 40px
    xl: 56px
    xxl: 96px
```

Although these shorthands are easy to understand, it’s cumbersome to add additional spacing values on a project, especially when values are needed later that fall between existing values.

Here’s an example on a recent project of mine:
```
  spacing:
    2xs: 4px
    xs: 8px
    sm: 16px
    md: 24px
    lg: 32px
    xl: 48px
    2xl: 64px
    3xl: 96px
    4xl: 128px
```

How do folks feel about switching to the [number-based spacing system](https://designsystem.digital.gov/design-tokens/spacing-units/) used in USWDS?

Example:
```
  spacing:
    0.5: 4px
    1: 8px
    1.5: 12px
    2: 16px
    2.5: 20px
    3: 24px
    4: 32px
    5: 40px
    6: 48px
    7: 56px
    8: 64px
    9: 72px
    10: 80px
    15: 120px
```

Since the numbers reflect the multiple of the base 1 value, we could simplify this to a single base spacing value, and update the `gesso-spacing()` function to return the appropriate value based on the multiple given.

**Comment by coreylafferty** [comment]
@dcmouyard I like this idea.

**Comment by dcmouyard** [comment]
After using the USWDS approach for awhile, I’m leaning towards getting rid of these spacing tokens and just having a single base spacing token _(probably set to 8px)_ and updating the `gesso-spacing` function to return a multiple of that base value. That way, if I need to use `gesso-spacing(11)` it will just work, and I won’t need to add `11: 88px` to the design tokens.

One counterpoint I can think of, though, is if designers really want to limit which spacing values we can use on a project.

**Comment by coreylafferty** [comment]
I've updated the key names for spacing and font sizes in this PR: https://github.com/forumone/gesso/pull/320

**Comment by coreylafferty** [comment]
Closed by #320
